test/nuts/suggestType/suggestType.nut.ts

```diff
@@ -66,6 +69,38 @@ describe('suggest types', () => {
     }
   });
 
+  it('it offers a suggestions on metadata format file when converting to metadata', async () => {
+    const lifecycleSpy = sinon.spy(Lifecycle.prototype, 'emitWarning');
+
+    const set = await ComponentSetBuilder.build({
+      sourcepath: [
+        path.join(
+          session.project.dir,
+          'force-app',
+          'main',
+          'default',
+          'enablementMeasureDefinitions',
+          'measure.enablementMeasureDefinition'
+        ),
+      ],
+    });
+    const converter = new MetadataConverter();
+    await converter.convert(set, 'metadata', {
+      type: 'directory',
+      genUniqueDir: false,
+      outputDirectory: 'output',
+    });
+    expect(lifecycleSpy.calledOnce).to.be.true;
```

Comment:
if this comes back `false` does that mean it was not called, or called twice?
[see below comment...does the warning count matter, or does the test only care that the expected warning happened?]

```suggestion
    expect(lifecycleSpy.callCount).to.equal(1);
```